traitsui/qt4/ui_panel.py

```diff
@@ -733,6 +761,9 @@ def _add_items(self, content, outer=None):
         show_left = group.show_left
         columns = group.columns
 
+        self._label_enabled_whens = []
+        self._label_visible_whens = []
```

Comment:
let's just make these private `List` traits on the class instead?

Reply:
It looks like `_GroupPanel` has `object` as a base class / is not a `HasTraits` class